app/views/shared/_sidebar.html.erb

```diff
@@ -3,6 +3,29 @@
     <ul>
       <li>
         <%= link_to "About", "/about", sidebar_link_options("about") %>
+        <ul class="<%= @section == "about" ? "expanded" : ""%>">
+          <li>
+            <%= link_to "Branching and Merging", "/about", sidebar_link_options("branching-and-merging") %>
+          </li>
+          <li>
+            <%= link_to "Small and Fast", "/about/small-and-fast", sidebar_link_options("small-and-fast") %>
+          </li>
+          <li>
+            <%= link_to "Distributed", "/about/distributed", sidebar_link_options("distributed") %>
+          </li>
+          <li>
+            <%= link_to "Data Assurance", "/about/info-assurance", sidebar_link_options("info-assurance") %>
+          </li>
+          <li>
+            <%= link_to "Staging Area", "/about/staging-area", sidebar_link_options("staging-area") %>
+          </li>
+          <li>
+            <%= link_to "Free and Open Source", "/about/free-and-open-source", sidebar_link_options("free-and-open-source") %>
+          </li>
+          <li>
+            <%= link_to "Trademark", "/trademark", sidebar_link_options("trademark") %>
```

Comment:
have you tried to run the application locally? If you do, you'll see this now returns an error